src/content/docs/en/guides/images.mdx

```diff
@@ -240,6 +240,22 @@ By default, the `<Image />` component will produce a `.webp` file.
 - a preset (`low`, `mid`, `high`, `max`) that is automatically normalized between formats.
 - a number from `0` to `100` (interpreted differently between formats).
 
+##### inferSize
+
+<p><Since v="4.4.0" /></p>
+
+Allows you to set the original `width` and `height` of a remote image automatically.
+
+By default, this value is set to `false` and you must manually specify both dimensions for your remote image. If used with an unauthorized remote image, a request will still be made to the image domain, so the dimensions can be fetched.
+
+Set `inferSize: true` to infer these values from the image content when fetched. This is helpful if you don't know the dimensions of the remote image, or if they might change:
+
+```astro mark="inferSize"
+---
+import { Image } from 'astro:assets';
+---
+<Image src="https://example.com/cat.png" inferSize alt="A cat sleeping in the sun.">
```

Comment:
I was going to ask whether explicitly setting true is necessary, or whether just adding the attribute is enough! (assuming the `={true}` wasn't deleted by mistake)

It seems like there are a few ways to set this attribute. Ideally, we'd like to demonstrate all of them, and make sure it's clear when each is properly used:

- `inferSize = {true}` or simply `inferSize` as a component attribute
- `inferSize: true` when used with `getImage()`

I don't think we should show `inferSize: true` in the text explanation if that's not how it's used in the `<Image />` component. We could say:

> Add the `inferSize` attribute to infer...
or
> Set `inferSize` to `true` to infer... (and let the example show the `={true}` syntax)
or
> Add `inferSize={true}` to the `<Image />` component (or just `inferSize`) to infer...

Do you think any of those work?

Reply:
Yeah I see what you mean, I went with:
> Add inferSize to the <Image /> component (or inferSize: true to getImage()) 

With the width property for <Image /> there are two different documented syntaxes ( using a `number` and using a `string` ) and it basically is just left to the user to figure it out, so I think just showing them adding the  `inferSize` attribute to <Image /> is good enough. It could be a matter of persona preference though, I personally think `inferSize` looks much better than `inferSize={true}`

Reply:
Looks good to me!

If we're going with the simpler attribute here, then I'll review our blog post for 4.4 and make sure that's what is shown!